ui/src/views/Dashboard.vue

```diff
@@ -12,57 +12,61 @@
-                      {{$t('dashboard.number_of_active_jail')}}:
-                      <span :title="configuration.JailStatus.length">
-                        {{configuration.JailStatus.length | humanFormat}}
-                      </span>
-                    </h3>
-                    <h3 class="col-xs-6 col-sm-4 col-md-3 col-lg-2" >
-                      {{$t('dashboard.number_of_enabled_jails')}}:
-                      <span :title="configuration.JailEnabled">
-                        {{configuration.JailEnabled | humanFormat}}
-                      </span>
-                    </h3>
-                </div>
-                <div class="row row-dashboar">
-                    <div class="col-xs-12 col-sm-8 col-md-6 col-lg-4 resources-panel">
-                        <div class="panel panel-default">
-                            <div class="panel-heading">
-                                <h3 class="panel-title">
-                                    <span class="icon-header-panel">
-                                        <span class="fa pficon-on-running "></span>
-                                    </span>{{$t('dashboard.list_active_jails')}}
-                                </h3>
-                            </div>
-                            <div class="panel-body">
-                                <div v-if="configuration.JailStatus.length >0" >
-                                    
-                                <div  v-for="jail in configuration.JailStatus">
-                                    <div class="row">
-                                        <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">{{jail}}</div>
-                                    </div>
-                                </div>
-                                </div>
-                                <div v-else> 
-                                    <div class="row">
-                                        <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">{{$t('dashboard.Fail2ban_is_probably_down')}}</div>
-                                    </div>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
+            <div class="row row-dashboard">
+              <div class="col-lg-4">
+                <h3 >{{$t('dashboard.number_bans_per_jail')}}</h3>
+                <div v-if="Object.keys(configuration.TotalBannedIP).length == 0" class="empty-piechart">
+                    <span class="fa fa-pie-chart"></span>
+                    <div>{{ $t('dashboard.empty_piechart_label') }}</div>
                 </div>
-            </div> 
+                <div v-else id="fail2ban-pie-chart"></div>
+              </div>
+              <div class="col-lg-4">
+                  <div class="container-fluid">
+                      <div class="row ">
+                          <h3 class="col-xs-6 col-sm-4 col-md-3 col-lg-4">
```

Comment:
```suggestion
                          <h3 class="col-xs-6 col-sm-4">
```
As a rule of thumb: the larger the screen, the smaller the number of the element (expressed in number of columns in a 12-columns layout). Since the element is four columns wide in both small and large screens, I guess it should be four columns on medium screens too.
`col-xs-6 col-sm-4 col-md-4 col-lg-4` should be ok, or even better, `col-xs-6 col-sm-4`, that is equivalent but more concise